lib/instrumental.js

```diff
@@ -47,21 +47,23 @@ function build_payload(metrics, time_stamp) {
 
   // increment item.count 200 121820381
   for(k in metrics.counters) {
+    var name = format_metric_name(k);
```

Comment:
If name was a var outside this loop, the hypothetical performance for many metrics goes… better.